tests/testdata/optimize_input_1.json

```diff
@@ -1,58 +1,60 @@
 {
-    "ems": {
```

Comment:
could you keep indentation=4 so it's easier to see what really changed

Reply:
hmm visual studio did that automatically. any idea how to prevent it?